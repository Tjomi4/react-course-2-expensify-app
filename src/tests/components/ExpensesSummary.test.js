/**
 * @jest-environment node
 */
import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';
import expenses from '../fixtures/expenses';

test('ExpensesSummary component should render corectly with 1 expense', () => {
  const wrapper = shallow(<ExpensesSummary expensesCount={1} expensesTotal={235}/>);
  expect(wrapper).toMatchSnapshot();
});

test('ExpensesSummary component should render corectly with multiple expense', () => {
  const wrapper = shallow(<ExpensesSummary expensesCount={5} expensesTotal={23500}/>);
  expect(wrapper).toMatchSnapshot();
});