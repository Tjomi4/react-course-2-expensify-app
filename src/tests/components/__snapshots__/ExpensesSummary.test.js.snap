// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExpensesSummary component should render corectly with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     total 
    $2.35
  </h1>
</div>
`;

exports[`ExpensesSummary component should render corectly with multiple expense 1`] = `
<div>
  <h1>
    Viewing 
    5
     
    expenses
     total 
    $235.00
  </h1>
</div>
`;
